projects/observability/src/shared/graphql/model/schema/metric/graphql-metric-aggregation.ts

```diff
@@ -11,6 +11,8 @@ export const enum GraphQlMetricAggregationPath {
 
 export interface GraphQlMetricAggregation {
   value: number;
+  upperBound?: number;
```

Comment:
Comment from @aaron-steinfeld on Draft PR on this about maybe using separate interfaces. Going to respond here.

In `observability-specificiation-builder.ts` in `extractFromServerData` we only return `MetricTimeseriesInterval[]`, so if we fetch `baselineSeries` and `series` together like we do we need to combine all of the values for each interval. Unless we want to change the signature of that method... which could be kind of big.